Apps/Playground/Scripts/experience.js

```diff
@@ -219,7 +220,33 @@ CreateBoxAsync().then(function () {
                     });
                 }
 
-                xr.baseExperience.enterXRAsync(vr ? "immersive-vr" : "immersive-ar", "unbounded", xr.renderTarget);
+                let sessionMode = vr ? "immersive-vr" : "immersive-ar"
+                if (hololens) {
+                    // Because HoloLens 2 is a head mounted display, its Babylon.js immersive experience more closely aligns to vr
+                    sessionMode = "immersive-vr";
+
+                    // Below is an example for enabling hand tracking. The code is not unique to HoloLens 2, and may be reused for other WebXR hand tracking enabled devices.
+                    // Create a mesh to display for each hand joint
+                    const jointMesh = BABYLON.SphereBuilder.CreateSphere("jointParent", { diameter: 1, }, scene);

```

Comment:
It's alright to leave this in, but joints will still be displayed even without passing in jointMesh. What passing in a mesh allows us to do is to customize the joints used (default joint is a grey sphere).

Reply:
If we want to show a bare minimum approach, we can get rid of the jointMesh part, and only pass xr.input to the features manager.

Reply:
sounds good